cisco_aci/datadog_checks/cisco_aci/data/conf.yaml.example

```diff
@@ -44,53 +87,173 @@ instances:
+    #
+    # tls_verify: true
+
+    ## @param tls_use_host_header - boolean - optional - default: false
+    ## If a `Host` header is set, this enables its use for SNI (matching against the TLS certificate CN or SAN).
+    #
+    # tls_use_host_header: false
+
+    ## @param tls_ignore_warning - boolean - optional - default: false
+    ## If `tls_verify` is disabled, security warnings are logged by the check.
+    ## Disable those by setting `tls_ignore_warning` to true.
     #
-    # timeout: 15
+    # tls_ignore_warning: false
 
-    ## @param min_collection_interval - integer - optional - default: 180
-    ## This check makes a lot of API calls
-    ## it could sometimes help to add a minimum collection interval
+    ## @param tls_cert - string - optional
+    ## The path to a single file in PEM format containing a certificate as well as any
+    ## number of CA certificates needed to establish the certificate's authenticity for
+    ## use when connecting to services. It may also contain an unencrypted private key to use.
     #
-    # min_collection_interval: 180
+    # tls_cert: <CERT_PATH>
 
-    ## @param ssl_verify - boolean - optional - default: true
-    ## Instruct the check to validate SSL certificates when connecting to aci_url.
+    ## @param tls_private_key - string - optional
+    ## The unencrypted private key to use for `tls_cert` when connecting to services. This is
+    ## required if `tls_cert` is set and it does not already contain a private key.
     #
-    # ssl_verify: true
+    # tls_private_key: <PRIVATE_KEY_PATH>
 
-    ## @param tags - list of key:value element - optional
-    ## List of tags to attach to every metric, event, and service check emitted by this integration.
+    ## @param tls_ca_cert - string - optional
+    ## The path to a file of concatenated CA certificates in PEM format or a directory
+    ## containing several CA certificates in PEM format. If a directory, the directory
+    ## must have been processed using the c_rehash utility supplied with OpenSSL. See:
+    ## https://www.openssl.org/docs/manmaster/man3/SSL_CTX_load_verify_locations.html
+    #
+    # tls_ca_cert: <CA_CERT_PATH>
+
+    ## @param headers - mapping - optional
+    ## The headers parameter allows you to send specific headers with every request.
+    ## You can use it for explicitly specifying the host header or adding headers for
+    ## authorization purposes.
+    ##
+    ## This overrides any default headers.
+    #
+    # headers:
+    #   Host: <ALTERNATIVE_HOSTNAME>
+    #   X-Auth-Token: <AUTH_TOKEN>
+
+    ## @param extra_headers - mapping - optional
+    ## Additional headers to send with every request.
+    #
+    # extra_headers:
+    #   Host: <ALTERNATIVE_HOSTNAME>
+    #   X-Auth-Token: <AUTH_TOKEN>
+
+    ## @param timeout - number - optional - default: 10
+    ## The timeout for accessing services.
+    ##
+    ## This overrides the `timeout` setting in `init_config`.
+    #
+    # timeout: 10
```

Comment:
Timeout changed to 10: https://github.com/DataDog/integrations-core/blob/14643dd21d134bca25526be366e16efc77a8c2cd/datadog_checks_base/datadog_checks/base/utils/http.py#L42